packages/block-editor/src/components/justify-toolbar/index.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * External dependencies
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { ToolbarGroup } from '@wordpress/components';
+import {
+	justifyLeft,
+	justifyCenter,
+	justifyRight,
+	justifySpaceBetween,
+} from '@wordpress/icons';
+import { __ } from '@wordpress/i18n';
+
+const navIcons = {
+	left: justifyLeft,
+	center: justifyCenter,
+	right: justifyRight,
+	'space-between': justifySpaceBetween,
+};
+
+export function JustifyToolbar( { onChange, value, popoverProps } ) {
+	const icon = value ? navIcons[ value ] : navIcons.left;
+
+	return (
+		<ToolbarGroup
+			icon={ icon }
+			popoverProps={ popoverProps }
+			label={ __( 'Change items justification' ) }
+			isCollapsed
+			controls={ [
```

Comment:
I think it would make sense to have these controls as default, but be able to override what controls we should show here. For example in `buttons` we don't have `space-between` and I'm not sure if there was a reason for this..

Or something like `allowedControls: [ 'left, 'right' ]? 🤔

Reply:
Implemented in 37ae8ad